gcloud/_apitools/buffered_stream.py

```diff
@@ -13,32 +13,32 @@ class BufferedStream(object):
     """Buffers a stream, reading ahead to determine if we're at the end."""
 
     def __init__(self, stream, start, size):
-        self.__stream = stream
-        self.__start_pos = start
-        self.__buffer_pos = 0
-        self.__buffered_data = self.__stream.read(size)
-        self.__stream_at_end = len(self.__buffered_data) < size
-        self.__end_pos = self.__start_pos + len(self.__buffered_data)
-
-    def __str__(self):
+        self._stream = stream
```

Comment:
I would've left it alone, but the private variables made writing tests too hard.